packages/roosterjs-editor-plugins/lib/plugins/Paste/Paste.ts

```diff
@@ -75,6 +76,8 @@ export default class Paste implements EditorPlugin {
             const trustedHTMLHandler = this.editor.getTrustedHTMLHandler();
             let wacListElements: Node[];
 
+            sanitizeHtmlColorsFromPastedContent(fragment, sanitizingOption);
```

Comment:
minor: suggest to move this below all the if...else blocks, to make it not looks like that "high priority"